src/atlas/getLookUpTable.m

```diff
@@ -19,7 +19,7 @@
 
   atlasDir = returnAtlasDir(atlasName);
 
-  if ~ismember(atlasName, {'wang', 'neuromorphometrics', 'anatomy_toobox'})
+  if ~ismember(atlasName, {'wang', 'neuromorphometrics', 'anatomy_toobox', 'visfAtlas'})
```

Comment:
```suggestion
  if ~ismember(lower(atlasName), {'wang', 'neuromorphometrics', 'anatomy_toobox', 'visfatlas'})
```

Reply:
that was causing some tests to fail because the test requested `visfatlas`

this way the string comparison is done on lower case which makes the user's (our) life easier